### DISCUSSION TITLE
should deletion (key d in normal mode) copy ?

### DISCUSSION THREAD

**Discussion by Lishen1** [helix-editor/helix#7806] [General] [answered]
I found that default behavior (copy on delete) is very uncomfortably because usually when i delete selection i going to place here content from buffer , but content replaced by deleted selection and i have  to use alt+d always that not comfortable

**Comment by CptPotato** [comment, marked as answer]
You can rebind your keys to accommodate that. i.e. swap `delete_selection` with `delete_selection_noyank`, see the [keymap docs](https://docs.helix-editor.com/master/keymap.html).

Also if you want to replace the selection with what you have currently yanked, use `R`.
